multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/Constants.java

```diff
@@ -26,6 +26,9 @@ public class Constants {
     public static final String MTA_BACKUP_NAMESPACE = "mta-backup";
     public static final String MTA_FOR_DELETION_PREFIX = "to-be-deleted";
 
+    public static final Long LONG_DEFAULT_NULL_VALUE = 0L;
```

Comment:
Could you set more meaningful name for this constant, current name does not explain what is used for.